README.md

```diff
@@ -217,3 +218,21 @@ See also [free database services](https://github.com/DmitryScaletta/free-databas
 ]
 ```
 </details>
+
+## Lade.io
+
+[Pricing](https://www.lade.io/pricing) | [Deployment](https://www.lade.io/docs/platform/cli)
+
+* Free services for 5 apps or databases
+* Free apps with 128MB RAM
+* Free databases and disks with 1GB of storage
```

Comment:
I think these lines are redundant because this information is already in the table above.
Information about free databases can be added to the next line along with the list of databases.

Reply:
Which information is helpful to include about free databases?